flink-end-to-end-tests/test-scripts/test_ha_per_job_cluster_datastream.sh

```diff
@@ -126,6 +125,8 @@ function run_ha_test() {
     local neededTaskmanagers=$(( (${PARALLELISM} + ${TASK_SLOTS_PER_TM_HA} - 1)  / ${TASK_SLOTS_PER_TM_HA} ))
     start_taskmanagers ${neededTaskmanagers}
 
+    wait_num_of_occurence_in_logs "Job .* is submitted." 1 "standalonejob"
+    JOB_ID=$(grep -o "Job .* is submitted." $FLINK_LOG_DIR/*standalonejob*.log* | sed 's/Job \(.*\) is submitted./\1/g')
```

Comment:
I would like to use the following script to get the job id.

`JOB_ID=$(grep -E -o 'Job [a-z0-9]+ is submitted' $FLINK_LOG_DIR/*standalonejob*.log* | awk '{print $2}')`